Src/FluentAssertions/Equivalency/Steps/EnumEqualityStep.cs

```diff
@@ -35,11 +36,11 @@ public class EnumEqualityStep : IEquivalencyStep
             switch (context.Options.EnumEquivalencyHandling)
             {
                 case EnumEquivalencyHandling.ByValue:
-                    HandleByValue(comparands);
+                    HandleByValue(comparands, context);
```

Comment:
⛏️ I would only pass in the reason if that's the only thing you need.

Reply:
> Master, I am honoured by your visit.
I was out walking my dog. Your function, why does it take BigCommonStruct instead of just x and y?
BCS is where we keep x and y, master.
Bark
He's hungry.
Shall I prepare him food?
No, just open the fridge, let him take what he wants.

https://twitter.com/tvaneerd/status/977257752820113413?lang=en